ibm/service/contextbasedrestrictions/resource_ibm_cbr_rule.go

```diff
@@ -68,7 +82,7 @@ func ResourceIBMCbrRule() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"attributes": &schema.Schema{
-							Type:        schema.TypeList,
+							Type:        schema.TypeSet,
```

Comment:
Should other types,  `contexts`, `tags`, `api_types`, change to `schema.TypeSet`  too? Those should be unordered collection of items